docs/source/index.rst

```diff
@@ -20,71 +22,100 @@ and IPython organizations.
 
+* `nbconvert <http://nbconvert.readthedocs.org/en/latest/>`__: convert dynamic
+  notebooks to static formats such as HTML, Markdown, LaTeX/PDF, and
+  reStrucuredText.
+* `nbformat <http://nbformat.readthedocs.org/en/latest/>`__: work with notebook
+  documents programmatically.
 
-* `nbconvert <http://nbconvert.readthedocs.org/en/latest/>`__: convert notebooks to static formats such as HTML, Markdown, LaTeX/PDF and reStrucuredText.
-* `nbformat <http://nbformat.readthedocs.org/en/latest/>`__: work with notebook documents programmatically.
 
-Education
----------
+Education Projects
+------------------
 
-The following projects are focused on addressing the pain points of using the Jupyter Notebook in educational settings.
+Jupyter Notebooks offer exciting and creative possibilities in education. The
+following projects are focused on supporting the use of Jupyter Notebook in
+a variety of educational settings.
 
-* `nbgrader <http://nbgrader.readthedocs.org/en/stable/>`__: tools for managing and grading notebook based assignments.
+* `nbgrader <http://nbgrader.readthedocs.org/en/stable/>`__: tools for
+  managing, grading, and reporting of notebook based assignments.
 
-Deployment
-----------
 
-These projects are being developed to support deploying the notebook in various contexts that include multiuser capabilities and secure/scalable cloud deployments.
+Deployment Projects
+-------------------
 
-* `jupyterhub <https://github.com/jupyter/jupyterhub>`__: multi-user notebook for organizations with plugglable authentication and scalability.
-* `jupyter-drive <https://github.com/jupyter/jupyter-drive>`__: store notebooks on Google Drive.
-* `nbviewer <https://github.com/jupyter/nbviewer>`__: share notebooks as static HTML on the web.
-* `tmpnb <https://github.com/jupyter/tmpnb>`__: temporary, transient notebooks in the cloud.
-* `dockerspawner <https://github.com/jupyter/dockerspawner>`__: deploy notebooks for jupyterhub inside docker containers.
-* `tmpnb-deploy <https://github.com/jupyter/tmpnb-deploy>`__: deployment tools for tmpnb.
+To serve a variety of users and use cases, these projects are being
+developed to support notebook deployment in various contexts, including
+multiuser capabilities and secure, scalable cloud deployments.
 
-Architecture
-------------
+* `jupyterhub <https://github.com/jupyter/jupyterhub>`__: multi-user notebook
+  for organizations with plugglable authentication and scalability.
+* `jupyter-drive <https://github.com/jupyter/jupyter-drive>`__: store notebooks
+  on Google Drive.
+* `nbviewer <https://github.com/jupyter/nbviewer>`__: share notebooks as static
+  HTML on the web.
+* `tmpnb <https://github.com/jupyter/tmpnb>`__: create temporary, transient
+  notebooks in the cloud.
+* `dockerspawner <https://github.com/jupyter/dockerspawner>`__: deploy
+  notebooks for `jupyterhub` inside Docker containers.
+* `tmpnb-deploy <https://github.com/jupyter/tmpnb-deploy>`__: deployment tools
```

Comment:
The selfish piece of tmpnb-deploy is that's it's more geared towards our own deployment and not composable pieces for tmpnb deployments in general. One can adapt things from them, as we've witnessed from people, but standalone they're pretty heavy on opinion as we've figured them out.

I'm not sure if this means that I'm saying we shouldn't include it on the list or if it should say "deployment tools for the tmpnb backing try.jupyter.org"